ca_cdk_constructs/storage/aurora_clone_refresh.py

```diff
@@ -239,14 +251,17 @@ def __init__(
             "AuroraClone",
             payload=TaskInput.from_object(rule_input),
             lambda_function=cluster_clone_lambda,
+            output_path="$.Payload",
```

Comment:
the output looks like

```json
{
  "ExecutedVersion": "$LATEST",
  "Payload": {
    "status": "clone-complete",
    "message": "The cluster xxxxxx was cloned to yyyyy, available at xyz.eu-west-1.rds.amazonaws.com"
  }
}
```

and this extracts just the payload bit so it can be passed along